app/src/androidTest/java/com/github/sdp/mediato/MyFollowersFragmentTest.java

```diff
@@ -69,6 +70,7 @@ public void setUp() throws ExecutionException, InterruptedException, TimeoutExce
     scenario.onActivity(activity -> {
       FragmentManager fragmentManager = activity.getSupportFragmentManager();
       MyFollowersFragment myFollowersFragment = new MyFollowersFragment();
+      activity.getMyProfileViewModel().setUsername("user_test_1");
```

Comment:
As discussed, it might be good to add the class name in the username for debugging purpose.

Reply:
Perfect